polkadot/node/subsystem-bench/src/subsystem-bench.rs

```diff
@@ -287,7 +210,6 @@ fn main() -> eyre::Result<()> {
 		.filter(Some("polkadot_overseer"), log::LevelFilter::Error)
 		.filter(None, log::LevelFilter::Info)
 		.format_timestamp_millis()
-		// .filter(None, log::LevelFilter::Trace)
```

Comment:
I think this was left intentionally for easier debugging

Reply:
It looks like a leftover :-)